### PR TITLE
rename some Image metrics

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

unify the name convention for image functions
part of #729 

## Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: torchmetrics/image/lpip_similarity.py [hunk 1]
```diff
@@ -155,3 +157,36 @@ def compute(self) -> Tensor:
...
+        >>> _ = torch.manual_seed(123)
+        >>> lpips = LPIPS(net_type='vgg')
+        >>> img1 = torch.rand(10, 3, 100, 100)
+        >>> img2 = torch.rand(10, 3, 100, 100)
+        >>> lpips(img1, img2)
+        tensor(0.3566, grad_fn=<SqueezeBackward0>)
+    """
+
+    @deprecated(
+        target=LearnedPerceptualImagePatchSimilarity, deprecated_in="0.7", remove_in="0.8", stream=_future_warning
```
Comment (ethanwharris) [inline torchmetrics/image/lpip_similarity.py:181]: Should there maybe be tests to make sure that the deprecated warnings work?

**Review by ethanwharris — COMMENTED** [review]
Looks good, small suggestion :smiley:

File: torchmetrics/image/lpip_similarity.py (hunk 1 above)
Comment (Borda) [reply torchmetrics/image/lpip_similarity.py:181]: it minds, but as we use the same package in PL and Flash I hope it works fine
I think that @stancld also tested it locally with all his renames...
In fact, you are asking if the deprecation really raises a warning, as there is still doctest to test the correct outputs :rabbit:

**Review by ethanwharris — APPROVED** [review]
LGTM :smiley:

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**